package java4.chainedException;

public class DAOException extends Exception {
	 
    public DAOException(String message, Throwable cause) {
        super(message, cause);
    }
}
